packages/bridging/src/BridgingSdk/tokenPriority.ts

```diff
@@ -0,0 +1,65 @@
+import { NATIVE_CURRENCY_ADDRESS, SupportedChainId } from '@cowprotocol/sdk-config'
+
+/**
+ * High-priority stablecoins registry (USDC and USDT)
+ * These tokens get the highest priority when selecting intermediate tokens
+ */
+export const HIGH_PRIORITY_TOKENS: Partial<Record<SupportedChainId, Set<string>>> = {
+  [SupportedChainId.MAINNET]: new Set([
+    '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', // USDC
+    '0xdac17f958d2ee523a2206206994597c13d831ec7', // USDT
```

Comment:
@shoom3301 isn't simpler to just have a sorted list if we want to override the list of the provider

This way, in the presence of USDC and USDT we can choose the one we believe we can give better prices for

Other heuristic is to check coingecko 24h volume in DEXes